### DISCUSSION TITLE
Stuck at TextPredictor

### DISCUSSION THREAD

**Discussion by LiangZai-He** [autogluon/autogluon#1680] [Q&A] [answered]
Hi~ When I used  TextPredictor to process the Book Price Prediction Data(https://auto.gluon.ai/stable/tutorials/tabular_prediction/tabular-multimodal-text-others.html#sec-tabularprediction-text-multimodal). I stucked at the 'Validation sanity check: 0%'.I also tried other datasets, still the same problem.(Validation sanity check stuck at 0)
Thank you~
![image](https://user-images.githubusercontent.com/85053944/165874147-f2a581e8-6a36-46cc-b151-d024adb53c47.png)

**Comment by sxjscience** [comment, marked as answer]
Would you share the system configuration?

```python
# Replace this code with the output of the following:
from autogluon.core.utils import show_versions
show_versions()
```

**Reply by LiangZai-He** [reply]
Thank you~
INSTALLED VERSIONS
------------------
date                 : 2022-04-30
time                 : 14:35:24.376535
python               : 3.8.10.final.0
OS                   : Windows
OS-release           : 10
Version              : 10.0.19043
machine              : AMD64
processor            : AMD64 Family 25 Model 80 Stepping 0, AuthenticAMD
num_cores            : 16
cpu_ram_mb           : 16252
cuda version         : None
num_gpus             : 1
gpu_ram_mb           : [3383]
avail_disk_size_mb   : None

autogluon.common     : 0.4.0
autogluon.core       : 0.4.0
autogluon.features   : 0.4.0
autogluon.tabular    : 0.4.0
autogluon.text       : 0.4.0
autogluon.vision     : 0.4.0
autogluon_contrib_nlp: None
boto3                : 1.22.1
catboost             : 1.0.5
dask                 : 2021.11.2
distributed          : 2021.11.2
fairscale            : 0.4.6
fastai               : 2.5.6
gluoncv              : 0.11.0
lightgbm             : 3.3.2
matplotlib           : 3.4.0
networkx             : 2.6.3
nptyping             : 1.4.4
numpy                : 1.22.2
omegaconf            : 2.1.2
pandas               : 1.3.5
PIL                  : 9.0.1
psutil               : 5.8.0
pytorch_lightning    : 1.5.10
ray                  : 1.7.0
requests             : 2.25.1
scipy                : 1.7.3
sentencepiece        : None
skimage              : 0.19.2
sklearn              : 1.0.2
smart_open           : 5.2.1
timm                 : 0.5.4
torch                : 1.9.0+cu111
torchmetrics         : 0.7.3
tqdm                 : 4.63.0
transformers         : 4.16.2
xgboost              : 1.4.2

**Reply by LiangZai-He** [reply]
Hi~ I solved the problem by restarting the computer (every few times training), and resetting to a larger virtual memory. I'm not sure if this fundamentally solves the problem.
